packages/react-core/src/components/Alert/Alert.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+
+import AlertIcon from './AlertIcon';
+import AlertBody from './AlertBody';
+import AlertAction from './AlertAction';
+import Button from '../Button/Button';
+
+export const alertType = ['success', 'danger', 'warning', 'info'];
+
+const propTypes = {
+  type: PropTypes.oneOf(alertType).isRequired,
+  action: PropTypes.shape(Button.propTypes),
+  ariaLabel: PropTypes.string,
+  isAriaHidden: PropTypes.bool,
```

Comment:
this should not be a prop.  We will want to intelligently decide this.  I think all icons will have this anyways.